fiaas_skipper/deploy/deploy.py

```diff
@@ -63,6 +65,9 @@ def deploy(self, namespaces=None):
                 LOG.exception("Failed to deploy %s in %s", deployment_config.name, deployment_config.namespace)
             time.sleep(self._deploy_interval)
 
+    def status(self):
```

Comment:
Now that this is moved to a different class, it feels wrong that the api needs to call through Deployer to get the status. The Deployer is no longer involved in getting the status, why should it be involved in the call chain?

Reply:
Agreed.